guide/daily-spending-wallet/requesting.md

```diff
@@ -183,15 +183,25 @@ A payment request with only standard data, like an amount and date, communicates
    image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings.png"
    retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings@2x.png"
    layout = "float-left-desktop -background -shadow"
-   caption = "Let users attach payment meta data to single-use payment requests."
+   caption = "Let users attach payment metadata to single-use payment requests."
    alt-text = "Screen showing single payment request settings."
    width = 250
    height = 541
 %}
 
 Users should have the option to attach a note for record-keeping and for the sender to read, tags, labels and any other metadata that may be relevant to give more context to the payment.
 
-This meta data should also be able to be [backed up]({{ "/guide/onboarding/backing-up-a-wallet/" | relative_url }}) in case the user loses access to their wallet and has to restore it.
+This metadata should also be able to be [backed up]({{ "/guide/onboarding/backing-up-a-wallet/" | relative_url }}) in case the user loses access to their wallet and has to restore it.
+
+#### Add a requester name with NameDesc
+
+You can convey the name of the user requesting the payment. Suppose the user is able to define their name in the settings of their wallet app. The app could then append the user's name to the beginning of the description field in a BOLT11 invoice. For example, if Alice makes an invoice with a memo that says "For design work", then the description in the BOLT11 invoice would become:
```

Comment:
```suggestion
You can convey the name of the user requesting the payment. Suppose the user is able to define their name in the settings of their wallet app, the app could then append the user's name to the beginning of the description field in a BOLT11 invoice. For example, if Alice makes an invoice with a memo that says "For design work", then the description in the BOLT11 invoice would become:
```

Reply:
This change would make the sentence grammatically incorrect.

Reply:
Hmm yes that is true.